content/en/serverless/step_functions/_index.md

```diff
@@ -41,10 +41,20 @@ You can view end-to-end traces for a single Step Function execution and its asso
 
 To get started, follow the [installation instructions][1].
 
+### Retry Step Functions from within Datadog
+
+You can use [redrive][4] to continue failed AWS Step Functions from the point of failure, without needing a complete state machine restart. View your [Step Functions][5] in Datadog to identify failed executions and click on the **Failed** tag to open a redrive modal.
```

Comment:
```suggestion
You can retry any execution that is being monitored or [redrive][4] executions to continue failed AWS Step Functions from the point of failure, without needing a complete state machine restart. This can be done directly from Datadog.

To take action on you [Step Functions][5] in Datadog, you can either use the invocations list on a Step Function side-panel and click on the **Failed** pill to open a redrive modal, or open the Step Function Trace Map to retry an execution or redrive a failed execution.


```

Reply:
Can we also add this screenshot? 
![image](https://github.com/user-attachments/assets/f39307c3-e50a-4199-8ebf-718f50f43671)